src/components/main/flight/FlightInfoScreen.js

```diff
@@ -239,265 +377,357 @@ export default class FlightInfoScreen extends React.Component {
+                    {arrCityIata}) via {airlineName}{" "}
+                  </Text>
+                </View>
+
+                {/* Arrival Departure */}
+                <View style={styles.itineraryView}>
+                  <View style={styles.itineraryViewItem}>
+                    <Text style={styles.itineraryLabel}>{depAirportName}</Text>
+                    <Text style={styles.itineraryAirportCode}>
+                      {departureIata}
+                    </Text>
+                  </View>
+
+                  <View style={styles.plane_icon_view}>
+                    <Ionicons name={"ios-airplane"} style={styles.plane_icon} />
+                  </View>
+
+                  <View style={styles.itineraryViewItem}>
+                    <View style={styles.fixFlex}>
+                      <Text style={styles.itineraryLabel}>
+                        {arrAirportName}
+                      </Text>
+                      <Text style={styles.itineraryAirportCode}>
+                        {arrivalIata}
+                      </Text>
+                    </View>
+                  </View>
+                </View>
+
+                {/* One Way or Two WAy*/}
+                <View style={styles.seatIndexView}>
+                  <View style={styles.seatIndexViewItem}>
+                    <Text style={styles.seatIndexLabel}>One Way</Text>
+                  </View>
+
+                  <View style={styles.switchView}>
+                    <Switch
+                      onChange={() => this.handleTripIndex()}
+                      value={this.state.tripIndex}
+                      trackColor={{
+                        false: "transparent",
+                        true: COLORS.sandy,
+                      }}
+                      thumbColor={COLORS.forestgreen}
+                    />
+                  </View>
+
+                  <View style={styles.seatIndexViewItem}>
+                    <Text style={styles.seatIndexLabel}>Two Way</Text>
+                  </View>
+                </View>
               </View>
-            </View>
-            {/*Flight Number*/}
-            <Text style={styles.smallBlueText}>{i18n.t('FLIGHT NUMBER')}</Text>
-            <Text style={styles.bigBlueText}>{flightChars} {flightNums}</Text>
-            <View style={styles.planeInfoText}>
-              {/*Departure and arrival city & IATA*/}
-              <Text style={styles.midGreyText}>{depCityName}({depCityIata}) {i18n.t('to')} {arrCityName}({arrCityIata}) </Text>
-              <Text style={styles.midGreyText}>{i18n.t('via')} {airlineName}</Text>
-            </View>
-            <View style={styles.buttonBarBottom}>
-              {/*Seat class selector*/}
-              <View style={[styles.leftGreenButton, { opacity: (this.state.seatIndex == 'Economy') ? 1 : 0.5 }]}>
+
+              {/* Tabs */}
+              <View style={styles.midInnerView}>
+                {/* Economy */}
```

Comment:
![image](https://user-images.githubusercontent.com/5898380/87975901-16c7b480-ca81-11ea-8fcd-1a2ad9e53f3f.png)

Please see if you can remove the corners of the selected box.